src/utils/archive.ts

```diff
@@ -72,7 +72,7 @@ export async function createMetadataTransaction(
       data: JSON.stringify({
         name: data.filename,
         size: getSizeBytes(data.content),
-        lastModifiedDate: data.timestamp,
+        lastModifiedDate: Math.round(data.timestamp / 1000),
```

Comment:
any chance we can make this an actual date? `lastModifiedDate` hints at something of type timestamp not of type `timetamp in seconds`